mlx-nn/src/lib.rs

```diff
@@ -15,6 +15,7 @@ mod activation;
 mod container;
 mod convolution;
 mod dropout;
+pub mod embedding;
```

Comment:
Maybe we should be consistent that all layers are re-exported to the crate root instead of their own separate mod?